src/main/java/com/epam/ta/reportportal/core/bts/handler/GetBugTrackingSystemHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.core.bts.handler;
+
+import com.epam.ta.reportportal.commons.ReportPortalUser;
+import com.epam.ta.reportportal.entity.integration.Integration;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+public interface GetBugTrackingSystemHandler {
+
+	Integration getEnabledProjectOrGlobalIntegrationByUrlAndBtsProject(ReportPortalUser.ProjectDetails projectDetails, String url,
```

Comment:
can we split logic of retriveing project and global integration.
it is according to a Single responsobility principle.
https://en.wikipedia.org/wiki/Single_responsibility_principle

Reply:
done